_includes/homepage_content.html

```diff
@@ -145,7 +145,7 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
               </div>
               <div class="col-md-12 order-md-1 text-md-left homepage-hero">
-                <img class="img-fluid shadow-lg" alt="bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.png">
+                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.svg">
```

Comment:
Why are image paths changed? [SVGs were replaced in favor of PNGs](https://github.com/bisq-network/bisq-website/pull/304) a short while ago because of a strange issue on macOS.

Reply:
ty